.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Would it make sense to assign this to someone by default? Either to triage or directly review it? What could be the best approach here?

Reply:
Sure I can assign myself to every bug for review